src/test/scala/uk/gov/nationalarchives/aggregate/processing/AggregateProcessingLambdaSpec.scala

```diff
@@ -281,4 +281,47 @@ class AggregateProcessingLambdaSpec extends ExternalServiceSpec {
         .withUrl("/graphql")
     )
   }
+
+  "handleRequest" should "upload a draft-metadata.csv to S3 containing the supplied metadata" in {
```

Comment:
Rename unit test slightly to: `... to S3 when supplied metadata present in uploaded metadata"`